app/views/content/shared/quotes/_quote_maths_rotna.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="quote-rotna">
+   <%= render Content::PhotoQuoteListComponent.new(
+      [
+        {
+          image: "static/images/content/subjects/rotna-roy.jpeg",
```

Comment:
Do we want to use an image here? We don't have an image on other pages and it is quite big (especially on mobile)

But happy for you to decide either way @joegibb @Sarah-DfE !